poc/storage-mongo/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func ListDatabases(client *mongo.Client) {
+	out, err := client.ListDatabases(context.TODO(), bson.M{})
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println("@@ List of databases @@")
+	for _, i := range out.Databases {
+		fmt.Println(i.Name)
+	}
+}
+
+func main() {
+	// Use the SetServerAPIOptions() method to set the Stable API version to 1
+	serverAPI := options.ServerAPI(options.ServerAPIVersion1)
+	opts := options.Client().ApplyURI("mongodb+srv://dipankar:1234@cluster0.zako1lr.mongodb.net/?retryWrites=true&w=majority").SetServerAPIOptions(serverAPI)
```

Comment:
create a `.env` file to store `MONGO_URI`.

Reference this: https://www.mongodb.com/docs/drivers/go/current/quick-start/#query-your-mongodb-cluster-from-your-application

Reply:
Got it it will be fixed once we make these more refined :)